qdrant_docs_mcp/libraries/qdrant-client-python.json

```diff
@@ -1,5 +1,23 @@
 {
     "name": "qdrant-client",
-    "github": "https://github.com/qdrant/qdrant-client",
-    "language": "python"
+    "github": "https://github.com/qdrant/qdrant",
+    "language": "python",
+    "config_file": {
+        "description": "the vector database",
+        "sources": [
+            {
+                "name": "qdrant-client",
+                "language": "python",
+                "version_by": {
+                    "version_type": "github_tags",
+                    "value": "https://github.com/qdrant/qdrant-client"
+                },
+                "url": "https://github.com/qdrant/landing_page",
+                "include_files": [
+                    "qdrant-landing/content/documentation/headless/snippets/**/*.md"
+                ],
+                "src_type": "snippets_dir"
+            }
+        ]
+    }
```

Comment:
this is temporarily before the mvp release, config_file should be in project repos, not in ours